src/components/MoneyRequestHeader.js

```diff
@@ -53,6 +67,8 @@ const MoneyRequestHeader = (props) => {
     const payeeAvatar = isExpenseReport
         ? ReportUtils.getWorkspaceAvatar(props.report)
         : ReportUtils.getAvatar(lodashGet(props.personalDetails, [props.report.managerEmail, 'avatar']), props.report.managerEmail);
+    const policy = props.policies[`policy_${props.report.policyID}`];
+    const shouldShowSettlementButton = !isSettled && (Policy.isAdminOfFreePolicy([policy]) || props.report.type === CONST.REPORT.TYPE.IOU);
```

Comment:
In the IOU case, we should check if the user is the payer, right? We don't want to show the button to the payee